html/rendering/non-replaced-elements/form-controls/resets.html

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
+<title>default style resets</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+@namespace url(urn:not-html);
+
+input, select, button, textarea {
+  letter-spacing: initial;
+  word-spacing: initial;
+  line-height: initial;
+  text-transform: initial;
+
+  text-indent: initial;
+  text-shadow: initial;
+}
+input, select, textarea {
+  text-align: initial;
+
+}
+input[type=reset i], input[type=button i], input[type=submit i], input[type=color i], button {
+  display: inline-block;
+}
+input[type=radio i], input[type=checkbox i], input[type=reset i], input[type=button i],
+input[type=submit i], input[type=color i], input[type=search i], select, button {
+  box-sizing: border-box;
+}
+textarea
+{
+  white-space: pre-wrap;
+}
+marquee {
+  text-align: initial;
+}
+table {
+  box-sizing: border-box;
+  border-spacing: 2px;
+  border-collapse: separate;
+  text-indent: initial;
+}
+</style>
+
+<div id="tests">
+ <input type="hidden">
+ <input type="text">
+ <input type="search">
+ <input type="tel">
+ <input type="url">
+ <input type="email">
+ <input type="password">
+ <input type="date">
+ <input type="month">
+ <input type="week">
+ <input type="time">
+ <input type="datetime-local">
+ <input type="number">
+ <input type="range">
+ <input type="color">
+ <input type="checkbox">
+ <input type="radio">
+ <input type="file">
+ <input type="submit">
+ <input type="image">
+ <input type="reset">
+ <input type="button">
+ <select><optgroup><option></select>
+ <select multiple></select>
+ <button></button>
+ <textarea></textarea>
+ <table></table>
+ <marquee></marquee>
+</div>
+
+<div id="refs"></div>
+
+<script>
+  const props = ['letter-spacing', 'word-spacing', 'line-height', 'text-transform', 'text-indent', 'text-shadow ', 'box-sizing', 'border-spacing', 'border-collapse', 'white-space'];
+ const refs = document.getElementById('refs');
+ for (const el of document.querySelectorAll('#tests *')) {
+   const clone = fakeClone(el);
+   refs.append(clone);
+   const testStyle = getComputedStyle(el);
+   const refStyle = getComputedStyle(clone);
+   for (const prop of props) {
+     test(() => {
+       assert_equals(testStyle.getPropertyValue(prop), refStyle.getPropertyValue(prop));
```

Comment:
Kinda over-complicated compared to just checking the value with and without `property: initial` matches, but I guess this is ok.

Reply:
This approach is generalizes to any value, not just `initial` though. My idea is to try to test most of the rendering section like this. See https://github.com/web-platform-tests/wpt/issues/5625#issuecomment-524086203

Reply:
Sounds good :)

FWIW, there's some failures when I run this test locally in both Chrome and Firefox (`<select multiple>` has `white-space: nowrap` for example).

I haven't checked WebKit, but if they match Blink it may be worth fixing this test before landing.

Reply:
Some _common_ failures, I mean.

Reply:
Fixed in 2fb46830f0d1fbb2fb78101a8d3d1f1bf06500a9